webui/react/package.json

```diff
@@ -7,6 +7,7 @@
     "npm": ">=8.0.0"
   },
   "scripts": {
```

Comment:
the package-lock diff size doesn't match up with the changes here. are you on npm 8? 

we want the lock file on
```
  "lockfileVersion": 2,
```

Reply:
sorry my node version keeps "slipping" to v14, I'll fix this